rewrite-xml/src/main/java/org/openrewrite/xml/marker/JavaType.java

```diff
@@ -0,0 +1,13 @@
+package org.openrewrite.xml.marker;
+
+import lombok.Value;
+import lombok.With;
+import org.openrewrite.marker.Marker;
+
+import java.util.UUID;
+
+@Value
+@With
+public class JavaType implements Marker {
```

Comment:
I think we should probably move this marker to `rewrite-core` and name it something like `TypeReference` and then possibly add a `language` property to it. This way `FindType`, `ChangeType` and other recipes can support it and can at the same time be independent of the language that the markers occur in.